### PR TITLE
Implement GPIO on TizenRT

### PR DESCRIPTION
**Pull request by hs0225**
- implement GPIO on TizneRT
- refactoring GPIO
  -- delete native binding sync function

IoT.js-DCO-1.0-Signed-off-by: Hosung Kim hs852.kim@samsung.com

### CODE REVIEW

File: src/js/gpio.js [hunk 1]
```diff
@@ -84,6 +82,8 @@ function GpioPin(configuration, callback) {
     } else {
       configuration.mode = defaultConfiguration.mode;
     }
+  } else {
+    configuration.mode = defaultConfiguration.mode;
```
Comment (chokobole) [inline src/js/gpio.js]: Please don't repeat `configuration.mode = defaultConfiguration.mode;` How about  doing like below?
```js
if (process.platform === 'nuttx' || util.isUndefined(mode)) {
   configuration.mode = defaultConfiguration.mode;
} else {
 ...
}
```

Comment (hs0225) [inline src/js/gpio.js]: Done.

**Comment by daeyeon** [comment]
LGTM

**Comment by chokobole** [comment]
LGTM

**Pull request merged**